src/loaders/mmd1_load.c

```diff
@@ -401,85 +425,102 @@ static int mmd1_load(struct module_data *m, HIO_HANDLE *f, const int start)
 			block.lines = hio_read8(f);
 		}
 
-		/* Sanity check--Amiga OctaMED files have an upper bound of 3200 lines per block. */
-		if (block.lines + 1 > 3200) {
-			D_(D_CRIT "invalid line count %d in block %d", block.lines + 1, i);
-			goto err_cleanup;
+		size = block.numtracks * (block.lines + 1) * (ver ? 4 : 3);
+
+		if (mmdc > 0) {
```

Comment:
is this not a boolean?

Reply:
It is. I think I wrote it like that to match up with the `ver` check below it, but I can change it.